### PR TITLE
Graph coverage

### PR DESCRIPTION
**Pull request by byorda-glb**
### Purpose

Increase code coverage of the Graph folder

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

Aaron Tang (@QilongTang)
Michael Kirschner (@mjkkirschner)

### FYIs

Alfredo Pozo (@alfredo-pozo)

### CODE REVIEW

**Comment by byorda-glb** [comment]
All tests passing in execution #105 of the SelfServe job

File: test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs [hunk 1]
```diff
@@ -49,5 +49,46 @@ public void DictionaryLinkTest()
...
+        [Category("UnitTests")]
+        public void PrintExpressionTest()
+        {
+            string listTestFolder = Path.Combine(TestDirectory, "core");
+            string angleFilePath = Path.Combine(listTestFolder, "Angle.dyn");
+
+            RunModel(angleFilePath);
+
+            NodeModel degreesToRadiansNode = CurrentDynamoModel.CurrentWorkspace.NodeFromWorkspace("dcd9c6c6-6350-4292-a553-c57a764504b4");
+            Assert.AreEqual("(Math.DegreesToRadians Number)", degreesToRadiansNode.PrintExpression());
```
Comment (mjkkirschner) [inline test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs:83]: hmm, I think this is another good candidate for obsoleting. S expressions were from the old scheme based interpreter for dynamo...

Comment (QilongTang) [reply test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs:83]: @mjkkirschner I see there are a couple instances of usage in Dynamo code, also it is currently marked as public virtual, so should be part of API. Would you like to start obsoletion in Dynamo v2.8?

Comment (aparajit-pratap) [reply test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs:83]: I agree, this should be obsoleted. No tests necessary.

Comment (aparajit-pratap) [reply test/DynamoCoreTests/Graph/Nodes/NodeModelTests.cs:83]: There are no real usages of this method, just recursive calls and definitions.

**Comment by RobertGlobant20** [comment]
@mjkkirschner  @QilongTang 
Could you please consider in your review the code that I just added (and removed) for the Connectors folder?.

Thanks
cc: @byorda-glb  @alfredo-pozo

File: src/DynamoCore/Graph/Connectors/ConnectorModel.cs [hunk 2]
```diff
@@ -215,29 +215,6 @@ protected override void SerializeCore(XmlElement element, SaveContext context)
         protected override void DeserializeCore(XmlElement nodeElement, SaveContext context)
```
Comment (QilongTang) [inline src/DynamoCore/Graph/Connectors/ConnectorModel.cs:215]: Since this fuction does not do anything now and is an override. Can we remove?

Comment (RobertGlobant20) [reply src/DynamoCore/Graph/Connectors/ConnectorModel.cs:215]: @QilongTang 
Visual Studio is reporting that it has 56 references, then I guess is not a good idea to update 56 places in about 20 different files (probably).

image

Comment (QilongTang) [reply src/DynamoCore/Graph/Connectors/ConnectorModel.cs:215]: @RobertGlobant20 Sure

**Pull request merged**